From 2304c336e1455230a50b9a2edadd5f477843ab85 Mon Sep 17 00:00:00 2001
From: OpenEmbedded <oe.patch@oe>
Date: Sun, 30 Sep 2018 18:43:32 +0100
Subject: [PATCH] cacheflush

Signed-off-by: OpenEmbedded <oe.patch@oe>
---
 arch/arm64/include/asm/cacheflush.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/include/asm/cacheflush.h b/arch/arm64/include/asm/cacheflush.h
index 2e5fb97..9e91c73 100644
--- a/arch/arm64/include/asm/cacheflush.h
+++ b/arch/arm64/include/asm/cacheflush.h
@@ -72,6 +72,7 @@ extern void __clean_dcache_area_poc(void *addr, size_t len);
 extern void __clean_dcache_area_pou(void *addr, size_t len);
 extern long __flush_cache_user_range(unsigned long start, unsigned long end);
 
+EXPORT_SYMBOL(__flush_dcache_area);
 static inline void flush_cache_mm(struct mm_struct *mm)
 {
 }
-- 
2.7.4

